lib/ios-generic-simulators.js

```diff
@@ -0,0 +1,10 @@
+export default {
+  'ipad simulator': [
+    ['0.0', 'iPad Retina'],
```

Comment:
Might be worth noting that these need to be strictly in order, or the parsing logic won't work.

Reply:
Right. Maybe I should just sort them

Reply:
I sort them now to be safe.